folium/plugins/time_slider_choropleth.py

```diff
@@ -19,6 +19,8 @@ class TimeSliderChoropleth(JSCSSMixin, Layer):
     styledict: dict
         A dictionary where the keys are the geojson feature ids and the values are
         dicts of `{time: style_options_dict}`
+    date_options: str, default "ddd MMM DD YYYY"
```

Comment:
later on it's called `date_format`, why not call it that here as well?

Reply:
To be consistent with the other plugins. `timestamped_geojson` and `timeline` also use `date_options`. I'm not so happy about the name

Reply:
That makes sense 👍